packages/node_modules/@ciscospark/widget-message/src/selector.js

```diff
@@ -50,7 +50,7 @@ const getSpace = createSelector(
   [getConversation, getUsers],
   (conversation, users) => {
     const space = conversation;
-    const currentUser = users.get('currentUser');
+    const currentUser = users.getIn(['byId', users.get('currentUserId')]);
```

Comment:
Seems like we could almost use a helper for getting the current user, I've seen this code a bunch.

Reply:
Yea. I this is an inconsistency... I actually inject a prop at the top for `currentUser` but here and there I forgot about it.